packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/ImsPortIdField.tsx

```diff
@@ -157,65 +157,71 @@ function ImsPortId({
+                    labelAppend={<EuiText size="m">{description} </EuiText>}
                     error={showInlineError ? errorMessage : false}
                     isInvalid={error}
                     id={id}
                     fullWidth
                 >
-                    <section className="node-port">
-                        <div className="node-select">
-                            <EuiFormRow
-                                label="Node"
-                                id={`${id}.node`}
-                                fullWidth
-                            >
-                                <ReactSelect<Option, false>
-                                    inputId={`${id}.node.search`}
-                                    name={`${name}.node`}
-                                    onChange={onChangeNodes}
-                                    options={nodeOptions}
-                                    placeholder={nodesPlaceholder}
-                                    value={nodeValue}
-                                    isSearchable={true}
-                                    isDisabled={
-                                        disabled ||
-                                        readOnly ||
-                                        nodeOptions.length === 0 ||
-                                        !!nodeSubscriptionId
-                                    }
-                                    styles={reactSelectInnerComponentStyles}
-                                />
-                            </EuiFormRow>
-                        </div>
-                        <div className="port-select">
-                            <EuiFormRow label="Port" id={id} fullWidth>
-                                <ReactSelect<Option<number>, false>
-                                    inputId={`${id}.search`}
-                                    name={name}
-                                    onChange={(selected) => {
-                                        onChange(selected?.value);
-                                    }}
-                                    options={portOptions}
-                                    placeholder={portPlaceholder}
-                                    value={portValue || null}
-                                    isSearchable={true}
-                                    isDisabled={
-                                        disabled ||
-                                        readOnly ||
-                                        portOptions.length === 0
-                                    }
-                                    styles={reactSelectInnerComponentStyles}
-                                />
-                            </EuiFormRow>
-                        </div>
-                    </section>
+                    {(!nodeValue && nodeSubscriptionId && (
+                        <EuiText>
+                            This Node subscription is missing. Please check the
```

Comment:
Needs translation here

Reply:
Well... yes and no. The field is surf specific and is set to be deprecated ie moved to surf specific. The translations currently available are the ones that are provided by the backend so there would need to be a translation added to the default translations file that would need to be removed ones the field is moved. So all and all beter to leave it untranslated for now I think

Reply:
This is a bit of a grey area, normally i would say that all user-facing texts should be translated using our translation mechanism.